todo_app/data/trello_items.py

```diff
@@ -0,0 +1,106 @@
+from flask import config, current_app as app
+from todo_app.data.todo_items import Item
+from flask.globals import request
+from flask.wrappers import Request, Response
+import requests
+
+
+def get_auth_params():
+    return {'key': app.config['TRELLO_API_KEY'], 'token': app.config['TRELLO_API_SECRET']}
+
+
+def build_url(endpoint):
+    return app.config['TRELLO_BASE_URL'] + endpoint
+
+
+def build_params(params={}):
+    full_params = get_auth_params()
+    full_params.update(params)
+    return full_params
+
+
+def get_boards():
+    params = build_params()
+    url = build_url('/members/me/boards')
+
+    response = requests.get(url, params = params)
+    boards = response.json()
+
+    return boards
+
+
+def get_board(name):
```

Comment:
I don't think anything uses this functionality, since we've already specified the board_id as a config variable